kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -129,11 +128,14 @@
 
   @import '~kolibri.styles.definitions';
 
-  .btn {
-    position: absolute;
-    top: 8px;
-    right: 21px;
-    z-index: 1;
+  .fullscreen-btn {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    width: 100%;
+    margin: 0;
+    text-align: right;
+    box-shadow: none;
```

Comment:
Possible to use the `KLabeledIcon` component and just align-right rather than flexbox?